Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: all clean uninstall install
+
+CC ?= gcc
+PREFIX ?= /usr/local
+CFLAGS += -g -Wall -Wextra -Wpedantic
+
+all: dollarskip
```

Comment:
the `dollarskip` target is missing

Reply:
That is correct.

That's by design.

Reply:
```
$ ls
LICENSE  Makefile  PKGBUILD  Pictures  README.md  dollarskip  dollarskip.c  install.sh
$ make clean
rm -f dollarskip
$ make
cc -g -Wall -Wextra -Wpedantic    dollarskip.c   -o dollarskip
$ make
make: Nothing to be done for 'all'.
$ touch dollarskip.c
$ make
cc -g -Wall -Wextra -Wpedantic    dollarskip.c   -o dollarskip
$
```

Reply:
ah, ok. I didn't know you could do that.